docs/source/markdown/podman-create.1.md

```diff
@@ -1121,13 +1121,11 @@ required for VPN, without it containers need to be run with the --network=host f
 Environment variables within containers can be set using multiple different options:  This section describes the precedence.
 
 Precedence Order:
```

Comment:
Suggestion: maybe we should add "from least to greatest" here to indicate what the actual ordering is.

Reply:
Great idea.